Bootstrap fails with .DS_Store files

**Describe the bug**
When I run `npm run bootstap` I get this error:

```
(node:91078) UnhandledPromiseRejectionWarning: Error: Cannot find module '/Users/bravomartin/WORK/DSI/mechanic/mechanic/packages/create-mechanic/function-examples/.DS_Store/dependencies.json'
```

**Expected behavior**
No Error

**Desktop (please complete the following information):**
 - OS: MacOs

**Additional context**
It looks like it's interpreting the `.DS_Store` file as a folder and wrongly assumes a `dependencies.json` file inside it.

--- a/packages/create-mechanic/updateVersions.js
+++ b/packages/create-mechanic/updateVersions.js
@@ -24,6 +24,7 @@ const path = require("path");
     const bases = fs
       .readdirSync(path.join(__dirname, directory))
       .filter((d) => d !== "index.js");
+
     for (const base of bases) {
       const dependenciesPath = path.join(
         __dirname,
@@ -31,7 +32,11 @@ const path = require("path");
         base,
         `dependencies.json`
       );
+      // skip if file doesn't exist
+      if (!fs.existsSync(dependenciesPath)) continue;
+
       const dependencies = require(dependenciesPath);
+
       for (const depType in dependencies) {
         for (const dep in dependencies[depType]) {
           if (dep.startsWith("@mechanic-design/")) {
@@ -42,6 +47,7 @@ const path = require("path");
           }
         }
       }
+
       fs.writeFileSync(
         dependenciesPath,
         JSON.stringify(dependencies, null, 2) + "\n"

